CHANGES.md

```diff
@@ -1,25 +1,23 @@
 # 2.14.0 - 29 May 2024
 
-This release fixes the `RelationDataContent.update` method to follow `dict.update` semantics, that is allow both updating with another dict, an iterable, keyword arguments or a mixture thereof.
-
 ## Features
 
-* feat: add a `__str__` to ActionFailed, for better unexpected failure output in https://github.com/canonical/operator/pull/1209
+* Add a `__str__` to ActionFailed, for better unexpected failure output (#1209)
```

Comment:
So sorry about these.

GitHub auto-generated sentense-cased our items and I lower-cased those manually... or did it? Now I'm not sure.

I then copy-pasted that to the change log.

Maybe we could agree on a single convention for both 🤔 

Looking through /releases, we've got a mix of sentence-cased bullet points without prefixes, like "Add this" and lower-case, prefixed, like "feat: add that" releases.